examples/interfaces/cw20-allowances/Cargo.toml

```diff
@@ -16,6 +16,7 @@ cosmwasm-std = { workspace = true, features = ["staking"] }
 cosmwasm-schema = { workspace = true }
 serde = { workspace = true }
 sylvia = { path = "../../../sylvia" }
+cw-orch = { workspace = true }
```

Comment:
This is a bit of a trick.

All the interfaces (or actually any sylvia crate) that will be imported by a `cw-orch`-enabled sylvia contract will need to have this dependency. No code changes, just the dependency. The macro exansion enabled by the `orch` feature ends up referencing `::cw_orch::...`. Maybe there is a way around it?

Only seems a problem if you import interface crates that are not in your repo